### ISSUE TITLE
display the Zcash Foundation's mission prominently

### ISSUE THREAD

**Issue by zookozcash** [ZcashFoundation/ZcashFoundation#59]
I'm writing a document to circulate to regulators (among many others), and I wanted to finish the sentence: "The Zcash Foundation's mission is…". So I googled it, and that took me to https://z.cash.foundation/, and I studied that page and didn't see anything that suited, so I clicked on "About", which took me to https://z.cash.foundation//about/, but then I didn't find the mission written there, and it seemed like a dead end, so I gave up.

**Comment by zookozcash** [comment]
Okay, now I'm going to un-give-up and go find it, perhaps by searching through github, but the purpose of this ticket is to suggest that we make the mission more prominent for people who are that impatient. Maybe it deserves to have a top-level item on the nav bar? Or maybe be the top of the About page.

**Comment by zookozcash** [comment]
For the record, the answer was to click on "Repository" on the top-level nav bar. If I were an average visitor, that probably wouldn't have occurred to me to try, because I would think of "Repository" as leading to source code, and I wasn't looking for source code. I clicked "Repository" and sure enough it took me to github. But, since I know that the some of the Foundation's docs are currently hosted in github, I looked there and found a file named [MISSION.md](https://github.com/ZcashFoundation/ZcashFoundation/blob/master/MISSION.md).

It is an excellent file, by the way! And the top of it turns out to be that sentence that I overlooked at the beginning of my search: "Our mission is to serve the users of the Zcash protocol and blockchain."

Okay, opening a new ticket for the next issue…

**Comment by zookozcash** [comment]
other issue: https://github.com/ZcashFoundation/ZcashFoundation/issues/60

**Comment by acityinohio** [comment]
Concomitant to my suggestion on #60, I can also update the site (the easiest fix is the About page) to more prominently reflect an updated Mission after #60 is resolved. 👍 Thanks for the feedback!

**Comment by acityinohio** [comment]
Updated site with appropriate links to Mission in about page and a better, more visible description.
